### DISCUSSION TITLE
Trible and Quadruple clicks - option to disable?

### DISCUSSION THREAD

**Discussion by dados** [pyscripter/SynEdit#60] [Ideas]
The triple and quadruple click behaviour is not needed in some cases and can be annoying if a user accidentally clicks more than intended.
Has anyone else experienced this?

Could it be possible to add property options to enable/disable these clicks ?
What do you think ?

**Comment by pyscripter** [comment]
This is standard editing behavior.  Can you disable it in Word or your browser for example?
